{{cookiecutter.project_slug}}/config/settings/common.py

```diff
@@ -89,9 +89,9 @@
 # MANAGER CONFIGURATION
 # ------------------------------------------------------------------------------
 # See: https://docs.djangoproject.com/en/dev/ref/settings/#admins
-ADMINS = (
-    ("""{{cookiecutter.author_name}}""", '{{cookiecutter.email}}'),
```

Comment:
maintain the triple quote marks, because if the `cookiecutter.author_name` is, for example, `O'Brien`, the code will break